hironx_moveit_config/test/test_hironx_moveit.py

```diff
@@ -58,7 +58,6 @@ def __init__(self, *args, **kwargs):
     @classmethod
     def setUpClass(self):
 
-        rospy.init_node("test_hironx_moveit")
```

Comment:
I see, [ROS_Client](https://github.com/start-jsk/rtmros_hironx/pull/458/files/12dbe4ed531ade2f75c1354a2b5b1e966a7d248e..c3f875fc87ef82030c02740c4523e64df74dde99#diff-68a1160ea0ff512aa6963f8383742dd4L62) is also a ROS node.